### PR TITLE
Doc: Add contributors and datasets endpoints

### PR DESCRIPTION
**Pull request by pbougue**
As a lot of people ask for already used datas, let's explain them how to search if we already have them.

### CODE REVIEW

File: documentation/slate/source/includes/apis.md [hunk 1]
```diff
@@ -20,6 +20,39 @@ The only arguments are the ones of [paging](#paging).
 | `coverage/{lon;lat}`                    | Information about a specific region |
 
 
+<a name="datasets"></a>Datasets
+-------------------------------
+
+Very simple endpoint providing the sets of data that are used in the given coverage.
```
Comment (antoine-de) [inline documentation/slate/source/includes/apis.md:32]: I think it is important to add some explanation on what is exactly a contributor and a dataset

Comment (pbougue) [reply documentation/slate/source/includes/apis.md:32]: Done

File: documentation/slate/source/includes/apis.md [hunk 2]
```diff
@@ -20,6 +26,48 @@ The only arguments are the ones of [paging](#paging).
...
+-------------------------------
+
+Very simple endpoint providing the contributors of data for the given coverage.
+
+A contributor is a data provider (typically a transport authority), and can provide multiple [datasets](#datasets).
+For example, the contributor Italian Railways will provide a dataset for the national train and some others for the regional trains.
+We will try to put them in the same [coverage](#coverage) so that we assemble them in the same journey search, using both.
+
+Very usefull to know which contributors are used in the datasets forming a coverage.
+
```
Comment (stifoon) [inline documentation/slate/source/includes/apis.md:60]: Could be nice to add a sentence about contributors feeding feedpublisher property on every response. With licence and etc.

Comment (TeXitoi) [reply documentation/slate/source/includes/apis.md:60]: First, it should be activated on prod...

Comment (pbougue) [reply documentation/slate/source/includes/apis.md:60]: I agree with both... Maybe it should be part of a new PR (or part of #1754 as it should allow feed_publisher in prod).
We merge this and make a new PR? @stifoon if you don't have the time to I can prepare it.

**Pull request merged**